index.js

```diff
@@ -18,3 +21,229 @@ closeBtn.addEventListener('click', closeButton);
+    technology1: 'html',
+    technology2: 'bootstrap',
+    technology3: 'ruby',
+    technology4: 'Ruby on rails',
+    button: 'see project',
+    btnCard1: 'See Live',
+    btnCard2: 'See Source',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    nameCard: 'Multi Post Stories',
+    number: 6,
+    Description:
+      "A daily selection of privately personalized reads; no account or sign-ups required,has been the industry's standard",
+    descriptionCard:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    descriptionGym:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaL",
+    technology1: 'html',
+    technology2: 'bootstrap',
+    technology3: 'ruby',
+    technology4: 'Ruby on rails',
+    button: 'see project',
+    btnCard1: 'See Live',
+    btnCard2: 'See Source',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    nameCard: 'Multi Post Stories',
+    number: 7,
+    Description:
+      "A daily selection of privately personalized reads; no account or sign-ups required,has been the industry's standard",
+    descriptionCard:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    descriptionGym:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaL",
+    technology1: 'html',
+    technology2: 'bootstrap',
+    technology3: 'ruby',
+    technology4: 'Ruby on rails',
+    technology5: 'css',
+    button: 'see project',
+    btnCard1: 'See Live',
+    btnCard2: 'See Source',
+  },
+];
+
+const cardsHtml = function () {
+  for (let index = 0; index < arrProject.length; index += 1) {
+    firstWork.innerHTML = ` <div class="work-1__detail">
+                <img
+                  src="./assets/icons/Img Placeholder.svg"
```

Comment:
- [ ] Good job with the removal of the HTML contents :smile: but I think the image paths need a keen check again. I can't see the `background-image` for the other 6 projects from my end here. Making this to work across all devices and browsers will really improve user experience on your page. I have a attached a screenshot below:
![image](https://user-images.githubusercontent.com/99715640/223857919-9bafd515-3f6a-4baf-ac02-2de8452a9487.png)

Reply:
Could you fix your browser, please? because everything is working right here, the problem is on your end.

Reply:
- **As you may have noticed, how the first card is vividly showing, this suggests that your relative path is not correctly set up. You can see it there probably because your browser has previously loaded it, so somehow, it's not requesting for that content again since it's not new. But this is not the case for first time visitors on your page.** Where the problem lies is in the naming convention of your img file. I think you need to escape any space in-between the name. Something like `<img href="./assets/images/Img\ Placeholders.png">`

Reply:
🤔 I just did on another browser now, and it's the same - on Firefox and Chrome.

- If you think it checks well now, you can fix the Pop up modal on bigger screens then request for another review.

Reply:
**Resolved**
Kindly clone the repository to see the changes made.


<img width="1440" alt="Screenshot 2023-03-09 at 10 16 19" src="https://user-images.githubusercontent.com/92754123/223978730-cb33f65e-f83e-4fca-95bf-a2afaf7b239b.png">